hudi-client/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -52,6 +52,8 @@
   private static final String TIMELINE_LAYOUT_VERSION = "hoodie.timeline.layout.version";
   private static final String BASE_PATH_PROP = "hoodie.base.path";
   private static final String AVRO_SCHEMA = "hoodie.avro.schema";
+  private static final String SCHEMA_CHECK = "hoodie.schema.check";
```

Comment:
rename to `hoodie.avro.schema.validate` ?

Reply:
Done